library/packages/src/lib/y2packager/licenses_handlers/rpm.rb

```diff
@@ -0,0 +1,90 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2018 SUSE LLC, All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+
+require "y2packager/licenses_handlers/base"
+
+Yast.import "Pkg"
+
+module Y2Packager
+  module LicensesHandlers
+    # This class is responsible for interacting with an rpm in order to get/set
+    # the license acceptance status for a given product
+    class Rpm < Base
+      NO_ACCEPTANCE_FILE = "no-acceptance-needed".freeze
+
+      # Determine whether the license should be accepted or not
+      #
+      # @return [Boolean] true if the license acceptance is required
+      def confirmation_required?
+        @package ||= find_package
+
+        return false unless @package
+
+        !ignore_acceptance?
```

Comment:
to be honest, I do not like the flow method have here. It requires finding package and ignore_acceptance already expect that package is found. What about having lazy load of package ( so not attr_reader ) and if it returns nil, then, return false and having package search here ( so ignore_acceptance method is here )?